packages/main/test/pages/Icon_and_theming.html

```diff
@@ -34,9 +34,83 @@
 	<ui5-icon name="tnt-v3/actor"></ui5-icon>
 
 	<br/><br/>
-	<div>BS Icons - single version ("business-suite/add-polygone")</div>
-	<ui5-icon class="samples-margin" name="business-suite/add-polygone"></ui5-icon>
+	<div>BS Icons - single version ("business-suite-v1/add-polygone")</div>
+	<ui5-icon class="samples-margin" name="business-suite-v1/add-polygone"></ui5-icon>
 
+	<div> BS Icons - couples' (v1 and v2) showcase of different icons</div>
+	<div class="couples-box">
+		<div class="two-icons-box">
+			<div class="version">v1</div>
+			<ui5-icon class="samples-margin" name="business-suite-v1/add-outside"></ui5-icon>
+		</div>
+
+		<div class="two-icons-box">
+			<div class="version">v2</div>
+			<ui5-icon class="samples-margin" name="business-suite-v2/add-outside"></ui5-icon>
+		</div>
+
+		<div class="two-icons-box">
+			<div class="version">v1</div>
+			<ui5-icon class="samples-margin" name="business-suite-v1/activate"></ui5-icon>
+		</div>
+
+		<div class="two-icons-box">
+			<div class="version">v2</div>
+			<ui5-icon class="samples-margin" name="business-suite-v2/activate"></ui5-icon>
+		</div>
+
+		<div class="two-icons-box">
+			<div class="version">v1</div>
+			<ui5-icon class="samples-margin" name="business-suite-v1/add-note"></ui5-icon>
+		</div>
+
+		<div class="two-icons-box">
+			<div class="version">v2</div>
+			<ui5-icon class="samples-margin" name="business-suite-v2/add-note"></ui5-icon>
+		</div>
+
+		<div class="two-icons-box">
+			<div class="version">v1</div>
+			<ui5-icon class="samples-margin" name="business-suite-v1/add-point"></ui5-icon>
+		</div>
+
+		<div class="two-icons-box">
+			<div class="version">v2</div>
+			<ui5-icon class="samples-margin" name="business-suite-v2/add-point"></ui5-icon>
+		</div>
+
+		<div class="two-icons-box">
+			<div class="version">v1</div>
+			<ui5-icon class="samples-margin" name="business-suite-v1/products"></ui5-icon>
+		</div>
+
+		<div class="two-icons-box">
+			<div class="version">v2</div>
+			<ui5-icon class="samples-margin" name="business-suite-v2/products"></ui5-icon>
+		</div>
+	</div>
+
+	<style>
+		.two-icons-box {
+			display: flex;
+			flex-direction: column;
```

Comment:
Also, you can look around and see the other test pages - there is no style tags any more as it's not CSP compliant - the styles are extracted in external files and added with link. You need to do the same if you want to keep these styles.
And, if you decide to have the styles, it's good to apply the same styles for the other examples above to have a constant test page